pub-matic-htb.js

```diff
@@ -72,8 +72,112 @@ function PubMaticHtb(configs) {
 
     /* Utilities
      * ---------------------------------- */
+     var __populateImprObject = returnParcels => {
```

Comment:
please change to es5 syntax

Reply:
changing it to following syntax:
function _populateImprObject (returnParcels) {}